libcap-ng/Regression/compare-capabilities-from-captest-and-capsh/runtest.sh

```diff
@@ -37,14 +37,14 @@ rlJournalStart
 	rlRun "uname -a"
         rlRun "TmpDir=\$(mktemp -d)" 0 "Creating tmp directory"
         rlRun "pushd $TmpDir"
-	rlRun "captest --text &> captest.out"
-	rlRun "captest &> hexadec.out"
+	rlRun "captest --text | tee captest.out"
+	rlRun "captest | tee hexadec.out"
```

Comment:
Is this replacement of the stdout+stderr forward just by the stdout pipe intentional?
I know it will help with debugging but now you catch just stdout.
Beakerlib has the `rlRun` option `-s` to catch both while still letting it got to the console. Then `$rlRun_LOG` may be used for grepping or asserting.

Reply:
Good point, fixed by using -s option of rlRun.